<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Input user login to turn off him or ban.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="userEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>On this page u can ban users and show users list.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="userView"/>
        </item>
        <item>
         <widget class="QPushButton" name="showusersButton">
          <property name="text">
           <string>show all users</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="offButton">
          <property name="text">
           <string>Turn off user</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="onButton">
          <property name="text">
           <string>Unban user</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="banUserButton">
          <property name="text">
           <string>Ban user</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopButton">
          <property name="styleSheet">
           <string notr="true">QPushButton{background-color:#ff0509;}</string>
          </property>
          <property name="text">
           <string>Stop server</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>On this page u can check all messages.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="messagesView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="publicButton">
            <property name="text">
             <string>Show all chat</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="privateButton">
            <property name="text">
             <string>Show private messages</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuclose">
    <property name="title">
     <string>menu</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionclose"/>
    <addaction name="actionwork_in_progress"/>
   </widget>
   <addaction name="menuclose"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionclose">
   <property name="text">
    <string>close</string>
   </property>
  </action>
  <action name="actionwork_in_progress">
   <property name="text">
    <string>work in progress....</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
